packages/artifact/src/internal/download/download-artifact.ts

```diff
@@ -128,10 +128,7 @@ export async function streamExtractExternal(
               if (!createdDirectories.has(path.dirname(fullPath))) {
                 createdDirectories.add(path.dirname(fullPath))
                 await resolveOrCreateDirectory(path.dirname(fullPath)).then(
-                  () => {
-                    entry.autodrain()
-                    callback()
-                  }
+                  () => {}
```

Comment:
you can drop the `.then( ... )` altogether

Reply:
Ah, you're right! Got my wires crossed from when I was using a promises array.

Reply:
https://github.com/actions/toolkit/pull/1670/commits/dcc55dfd0485545474048b888c6167aa9206d92c